src/main/resources/bootstrap.yaml

```diff
@@ -0,0 +1,23 @@
+spring:
+  cloud:
+    propertiesvolume:
+      prefixed: false
+      paths: /mnt/secrets/rpe-ft-api
+      aliases:
+        admin-username-cmc: ADMIN_USERNAME_CMC
```

Comment:
are these aliases needed?

Reply:
I have tried in my local and it doesn't work without them as they are being referenced in application,yaml explicitly as ADMIN_USERNAME_CMC.
https://github.com/hmcts/feature-toggle-api/blob/a68b543ff2687de8db0644746e2f9de23881a2de/src/main/resources/application.yaml#L42